tests/resources/Engine.py

```diff
@@ -1824,3 +1824,42 @@ def modify_retention_dat(poller, host, service, key, value):
             f"{VAR_ROOT}/log/centreon-engine/config{poller}/retention.dat", "w")
         ff.writelines(lines)
         ff.close()
+
+
+def change_engine_log_level(domain: str, level: str):
+    f = open(f"{CONF_DIR}/config0/centengine.cfg", "r")
+    lines = f.readlines()
+    f.close
+    r = re.compile(f"^log_level_{domain}=[a-z]+$")
+    new_lines = [f"log_level_{domain}={level}\n" if r.match(
+        line) is not None else line for line in lines]
+    if new_lines != lines:
+        f = open(f"{CONF_DIR}/config0/centengine.cfg", "w")
+        f.writelines(new_lines)
+        f.close()
+        return
+    f = open(f"{CONF_DIR}/config0/centengine.cfg", "w")
+    lines.append(f"log_level_{domain}={level}\n")
+    f.writelines(lines)
+    f.close()
+
+
+def change_engine_command(command_index: str, new_command: str):
+    f = open(f"{CONF_DIR}/config0/commands.cfg", "r")
+    lines = f.readlines()
+    f.close
+    new_lines = []
+    r = re.compile(f"^\\s+command_name\\s+command_{command_index}$")
+    found = 0
+    for line in lines:
+        if found == 1:
+            found = 0
+            new_lines.append(
+                f"    command_line                    {new_command}\n")
+        else:
+            new_lines.append(line)
+        if r.match(line) is not None:
+            found = 1
+    f = open(f"{CONF_DIR}/config0/commands.cfg", "w")
+    f.writelines(new_lines)
+    f.close
```

Comment:
There is already a function named engine_config_set_value in Engine.py that can do the job in these two cases.

Reply:
done for the first case not for commands.cfg